<?xml version="1.0"?>

<launch>
  <node launch-prefix="nice" name="bluefox_cam" pkg="bluefox_cam" type="bluefox_cam" output="screen">
    <param name="device_num" value="0" />
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <param name="camera_frame_id" value="bluefox_cam" />
    <param name="output_stat" value="1" />
    <remap from="/bluefox_cam/image_raw" to="/usb_cam/image_raw"/>
  </node>
  
</launch>
